Exec/RegTests/EB-ConvergingNozzle/CMakeLists.txt

```diff
@@ -2,9 +2,4 @@ set(PELEC_ENABLE_PARTICLES OFF)
 set(PELEC_EOS_MODEL GammaLaw)
 set(PELEC_CHEMISTRY_MODEL Null)
 set(PELEC_TRANSPORT_MODEL Constant)
-
-#Generic stuff for every executable
-get_filename_component(DIR_NAME ${CMAKE_CURRENT_SOURCE_DIR} NAME)
-set(pelec_exe_name PeleC-${DIR_NAME})
-include(BuildPeleCExe)
-build_pelec_exe(${pelec_exe_name})
+include(BuildExeAndLib)
```

Comment:
Can we check to see if all this stuff is also done in the `Production` cases?

Reply:
Do we want to build the Production cases? It would cancel out the speedup from this PR.

Reply:
I guess maybe we should add these in. I can do so in a PR after this.